samples/createJobWithStaticOverlay.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright 2021, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
nit: we usually hyphen case our Node.js samples.

Reply:
Ack. Do I need to change this? I will then have to rename every .js file in the samples.